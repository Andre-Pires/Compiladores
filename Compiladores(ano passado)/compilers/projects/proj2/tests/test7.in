float exp(float x, int e){
  int i;
  float n_exp_e;
  i=1;
  n_exp_e=1;
  while(i<=e)
    n_exp_e*=x;
  return n_exp_e;
}
