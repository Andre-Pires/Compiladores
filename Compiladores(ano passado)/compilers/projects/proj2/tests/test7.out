warning: type cast int to float, line 5
